siatest/renter/renter_test.go

```diff
@@ -75,3 +79,49 @@ func testUploadDownload(t *testing.T, tg *siatest.TestGroup) {
 		t.Error(err)
 	}
 }
+
+// testDownloadMultipleLargeSectors downloads multiple large files (>5 Sectors)
+// in parallel and makes sure that the downloads are blocking each other.
+func testDownloadMultipleLargeSectors(t *testing.T, tg *siatest.TestGroup) {
+	// parallelDownloads is the number of downloads that are run in parallel.
+	parallelDownloads := 50
+	// fileSize is the size of the downloaded file.
+	fileSize := int(10 * modules.SectorSize)
+	// uniqueRemoteFiles is the number of files that will be uploaded to the
+	// network. Downloads will choose the remote file to download randomly.
+	uniqueRemoteFiles := 5
+	// Grab the first of the group's renters
+	renter := tg.Renters()[0]
+
+	// Upload files
+	dataPieces := uint64(len(tg.Hosts())) - 1
+	parityPieces := uint64(1)
+	remoteFiles := make([]*siatest.RemoteFile, 0, uniqueRemoteFiles)
+	for i := 0; i < uniqueRemoteFiles; i++ {
+		remoteFile, err := renter.UploadNewFileBlocking(fileSize, dataPieces, parityPieces)
```

Comment:
can we make fileSize a function of `i`? Perhaps `modules.SectorSize + i*modules.SectorSize`?